### ISSUE TITLE
Just in case you have problems running this code!

### ISSUE THREAD

**Issue by gszswork** [whr000001/DELL#5]
Hi everyone,

First thank to the author with the great contribution of this work. I reproduce the experiment on my own datasets and am publishing. During it, I encounter some minor issues like missing implementation or bugs. Here based on my hand-on experience, I made a updated version of this repo with the following changes: 

1. Fix the training split under expert as train:valid:test = 7:1:2, remove the usage of 3 split files.
2. According to the available proxy, only utilize the remaining available proxy tasks (content.py, search_entity_from_wiki.py, retrieval.py, stance.py, response.py) for fake news detection.
3. Fix bugs in entity/wiki proxy (Some news has None entities returned).
4. Optimize the save_state_dict() and load_state_dict() of ./expert/train.py, filtered out the fixed-param tokenizer and lm.
5. Provide details on the running commands. 

Further discussion is always welcomed. Link: 

https://github.com/gszswork/DELL

Best regards

**Comment by whr000001** [comment]
Hi,

Thanks for your interest and thoughtful contributions to this work. We will add the link in the readme file.

Meanwhile, here are some of my thoughts:
1. We used three split files to shuffle the dataset and obtain a fair comparison with the baselines. The purpose of shuffling the dataset is to ensure that the label distributions of the training and test sets are similar. Thus, I think a better way is to use a fixed random seed and shuffle *dataset_ids*.
2. We believe and have experimentally proved that the proxy task of framing detection or propaganda technology detection could enhance misinformation/fake news detection performance. Thus, I think a better way is to set the proxy tasks as optional.

Sincerely
